traveler-ios-ui/TravelerKitUI/ViewControllers/CancellationReasonViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  CancellationReasonViewController.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-11-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+public protocol CancellationReasonViewControllerDelegate: class {
+    func cancellationReasonViewController(_ controller: CancellationReasonViewController, didCancel order:Order)
+    func cancellationReasonViewControllerDidExpire(_ controller: CancellationReasonViewController)
+}
+
+open class CancellationReasonViewController: UITableViewController {
+    @IBOutlet weak var explanationInputTextField: UITextField!
+
+    var quote: CancellationQuote?
+    weak var delegate: CancellationReasonViewControllerDelegate?
+
+    private var explanation: String?
+    private var selectedReasonIndexPath: IndexPath?
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    // MARK: UITableViewDataSource
+
+    override public func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return quote!.cancellationReasons.count
+    }
+
+    override public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cancelReasonCell", for: indexPath) as! InfoCell
+        cell.titleLabel.text = quote!.cancellationReasons[indexPath.row].value
```

Comment:
It would be better to rename the attribute value into "description". Value (IMO) suggests numbers, and the attribute's description says that it is the text representation of the cancellation reason. As such, "description" makes more sense.